packages/rocketchat-message-attachments/client/renderField.js

```diff
@@ -1,7 +1,37 @@
 const renderers = {};
 
-export function registerFieldTemplate(fieldType, templateName) {
+/**
+ * The field templates will be rendered non-reactive for all messages by the messages-list (@see rocketchat-nrr)
+ * Thus, we cannot provide helpers or events to the template, but we need to register this interactivity at the parent
+ * template which is the room. The event will be bubbled by the Blaze-framework
+ * @param fieldType
+ * @param templateName
+ * @param helpers
+ * @param events
+ */
+export function registerFieldTemplate(fieldType, templateName, events) {
 	renderers[fieldType] = templateName;
+
+	//propagate helpers and events to the room template, changing the selectors
+	// loop at events. For each event (like 'click .accept'), copy the function to a function of the room events.
+	// While doing that, add the fieldType as class selector to the events function in order to avoid naming clashes
+	if (events != null) {
+		const uniqueEvents = {};
+		//rename the event handlers so they are unique in the "parent" template to which the events bubble
+		for (const property in events) {
+			if (events.hasOwnProperty(property)) {
+				const event = property.substr(0, property.indexOf(' '));
+				const selector = property.substr(property.indexOf(' ') + 1);
+				Object.defineProperty(uniqueEvents,
+					`${ event }` + ` .${ fieldType } ` + `${ selector }`,
```

Comment:
You don't need the concatenation of multiple templates: `${ event } .${ fieldType } ${ selector }` should do

Reply:
changed as mentioned.